### PR TITLE
Edit list command message to be less wordy

### LINKED ISSUE #509: [PE-D] Error message for list command too wordy
Perhaps some content can be cut down for a more concise yet useful error message/ use a different colour to distinguish different points eg for command `show 1`
￼

-------------
Labels: `severity.VeryLow` `type.FeatureFlaw`
original: candyhy/ped#7

### PR DESCRIPTION
**Pull request by tlylt**
Fixes #509
image

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ In the example above, <span class="main-command">add</span> is the command word
 
 Here are some general rules to follow when entering prefixes and parameters:
 * A whitespace must be included before every prefix.<br>
-  e.g. `-x METHOD-u URL` is not acceptable, and ` -x METHOD -u URL` is in the correct format.<br>
+  e.g. `-x METHOD-u URL` is not acceptable, and `-x METHOD -u URL` is in the correct format.<br>
```
Comment (JulietTeoh) [inline docs/UserGuide.md:119]: thanks!

**Review by JulietTeoh — APPROVED** [review]
LGTM!

**Review by tjtanjin — APPROVED** [review]
LGTM :)
